e2e-tests/tests/helpers.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { expect } from "@playwright/test";
```

Comment:
I've seen these put into Page Object Models in other playwright code bases. This is a fairly minimal doc with an explanation: https://playwright.dev/docs/pom This tutorial is much better (but longer): https://ray.run/blog/mastering-poms

What do you think about follow that convention?

Reply:
Cool, make sense. Will update the PR.

Reply:
Done